src/queryUtils.ts

```diff
@@ -47,3 +49,9 @@ export const transformBulkResults = (results: Record[], query: string): SoqlQuer
     query,
   };
 };
+
+export const logFn = <T>(i: T): T => {
+  // eslint-disable-next-line no-console
+  console.error(i);
+  return i;
+};
```

Comment:
this isn't used in the plugin.

Reply:
yeah, I'll take it out.  I probably need to put it in `kit`.